tests/conftest.py

```diff
@@ -49,6 +50,19 @@ def fx_tmpdir(tmpdir):
 
 
 @yield_fixture
+def fx_change_cwd_to_python_dir():
```

Comment:
hmm. I find [what is `cd`](http://ss64.com/bash/cd.html) and it mean this

> Change Directory - change the current working directory to a specific Folder.

I think `change_cwd` and `cd`, both is good. So I have no idea about this issue.